packages/twenty-server/src/engine/metadata-modules/field-metadata/interfaces/relation-type.interface.ts

```diff
@@ -0,0 +1,6 @@
+export enum RelationType {
+  ONE_TO_ONE = 'ONE_TO_ONE',
+  ONE_TO_MANY = 'ONE_TO_MANY',
+  MANY_TO_ONE = 'MANY_TO_ONE',
+  MANY_TO_MANY = 'MANY_TO_MANY',
```

Comment:
MANY_TO_MANY won't exist in the current vision
(it's two ONE_TO_MANY + a layout option to not display the join table)